docs/deployment/provision/provider.mdx

```diff
@@ -8,15 +8,31 @@ Provider provisioning in Keep allows you to set up and manage data providers dyn
 
 ### Configuring Providers
 
-To provision providers, set the `KEEP_PROVIDERS` environment variable with a JSON string containing the provider configurations. Here's an example:
+To provision providers and deduplication rules for them, set the `KEEP_PROVIDERS` environment variable. This can be done in two ways:
```

Comment:
maybe mention that dedup rule is optioanl not mandatory

Reply:
Done